tests/arch/arm/arm_irq_vector_table/src/arm_irq_vector_table.c

```diff
@@ -230,6 +230,18 @@ vth __irq_vector_table _irq_vector_table[] = {
 	isr0, isr1, isr2, 0,
 	rtc_isr
 };
+#elif defined(CONFIG_SOC_SERIES_IMX_RT6XX)
```

Comment:
I think this should be:
```
#elif defined(CONFIG_SOC_SERIES_IMX_RT6XX) && defined(CONFIG_MCUX_OS_TIMER)
```
To account for the possibility that someone switches back to SYSTICK.

Reply:
Thank you. Done